dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/AppHubController.java

```diff
@@ -66,20 +71,32 @@
     @Autowired
     private I18nManager i18nManager;
 
-
-    @RequestMapping( method = RequestMethod.GET, produces = "application/json" )
-    public @ResponseBody List<WebApp> listAppHub( HttpServletResponse response )
+    /**
+     * Deprecated as of version 2.35 and should be removed eventually.
+     */
+    @GetMapping( produces = "application/json" )
+    public  List<WebApp> listAppHub( @RequestParam String query, HttpServletResponse response )
```

Comment:
I think this `@RequestParam String query` is new and shouldn't have been added?  It causes `/api/appHub` to fail with a 400 error when it should be an alias for `/api/appHub/v1/apps` I think?  Maybe it should automatically pass the `dhis2_version` parameter?  Not sure this is actually in use anywhere, but as-is this is an unnecessary breaking change

Reply:
Yep, this was a unintentional diff.